### ISSUE TITLE
Release v1.30.11

### ISSUE THREAD

**Issue by deckhouse-BOaTswain** [deckhouse/deckhouse#858]
:point_right: Use this issue to test milestone [v1.30.11](https://github.com/deckhouse/deckhouse/milestone/18) and deploy released tag.
            :point_right: Use 'e2e/run/' labels to run default e2e test.
            :point_right: Use 'e2e/use/' labels to run specific e2e test.
            :point_right: Use 'deploy/' labels to deploy site and documentation.
            :point_right: Use 'deploy/deckhouse/' labels to deploy to channels after creating tag.

**Comment by deckhouse-BOaTswain** [comment]
New commit [a40141](https://github.com/deckhouse/deckhouse/commit/a40141a0bfe370d72bf3fc079bbc6fe4162aa95d) in branch 'main':
```
[docs] Fix nodeUser CRD (#860)
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [ffac72](https://github.com/deckhouse/deckhouse/commit/ffac7204060b2598111de3a1dbc53de494e298f1) in branch 'main':
```
[bashible] fix centos distro version detection (#857)
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [8a444c](https://github.com/deckhouse/deckhouse/commit/8a444cff05a4a3ea46dd36ef2495f855882b5454) in branch 'main':
```
Add enviroment variables to CSI (#864)
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [aad26f](https://github.com/deckhouse/deckhouse/commit/aad26f9c3353ba583ddd5a15267f18de63aea8a5) in branch 'main':
```
[node-manager] add error file when bashible context failed (#850)

Signed-off-by: Yuriy Losev <yuriy.losev@flant.com>
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [a8f535](https://github.com/deckhouse/deckhouse/commit/a8f5356f5b9d271f732369066bdc94c48b5163f2) in branch 'main':
```
[candi] Support astra 1.7 (#847)


Co-authored-by: Denis Romanenko <denis.romanenko@flant.com>
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [6d8d4e](https://github.com/deckhouse/deckhouse/commit/6d8d4ec141ec6ae902cf26598788e5870f9a42d8) in branch 'main':
```
[docs] Fix article text in «The deckhouse module» page (#817)

[docs] Review «The deckhouse module» article

Co-authored-by: Artem Kladov <artem.gen.kladov@gmail.com>
Co-authored-by: Kirill Kononovich <41591254+kirkonru@users.noreply.github.com>
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [4d73e1](https://github.com/deckhouse/deckhouse/commit/4d73e17274196ef254f5ba2ae319e46fd5d923bd) in branch 'main':
```
Changelog schema template and link (#855)
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [b2f404](https://github.com/deckhouse/deckhouse/commit/b2f4042d528946132b1c3104d985c04dfef8627c) in branch 'main':
```
[ci] fix: use tagged dev/install imgae to run in e2e tests (#812)
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [30840c](https://github.com/deckhouse/deckhouse/commit/30840c648af93d0e26ea5e9547d2dbdef5090246) in branch 'main':
```
[dhctl][deckhouse] Control plane node readiness check before converge node (#353)
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [25bb00](https://github.com/deckhouse/deckhouse/commit/25bb00e371a257c2983125d07196004551871dfe) in branch 'main':
```
[docs] Add warning about viewing an unreleased version (#873)

[docs] Add warning about viewing an unreleased version
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [553055](https://github.com/deckhouse/deckhouse/commit/5530556ed129d573f9cc1d3db5055acd74bf1237) in branch 'main':
```
ci: enable dry-run mode for werf-cleanup (#885)

Signed-off-by: Timofey Kirillov <timofey.kirillov@flant.com>
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [9ba8ba](https://github.com/deckhouse/deckhouse/commit/9ba8ba8e75e3ad58b77c81eca3ced5d70d5dc3ed) in branch 'main':
```
[dhctl] Create additional kube resources in order (#833)
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [467959](https://github.com/deckhouse/deckhouse/commit/4679597dc8d6433e6907b9b7d4ef4b9b1f250dcf) in branch 'main':
```
Fix changelog job condition (#882)
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [652104](https://github.com/deckhouse/deckhouse/commit/65210430e47187ecf745e05ec72d532b23131c5a) in branch 'main':
```
[node-manager] sort node users by name (#888)

Signed-off-by: Yuriy Losev <yuriy.losev@flant.com>
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [9b32aa](https://github.com/deckhouse/deckhouse/commit/9b32aa43ce36121db27ae54548b8d654d8cc6873) in branch 'main':
```
[node-manager] Fix constant OOM on small-sized nodes (#876)

Co-authored-by: Denis Romanenko <denis.romanenko@flant.com>
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [b5ca53](https://github.com/deckhouse/deckhouse/commit/b5ca530453d8b48dba9d899f5addf796b21571ab) in branch 'main':
```
[candi] proper work with astra bundle in EE/FE (#868)
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [f7a45a](https://github.com/deckhouse/deckhouse/commit/f7a45aa44a19c3559f78258a9aa87ba0320ca589) in branch 'main':
```
Re-generate changelog (#859)

Co-authored-by: deckhouse-BOaTswain <deckhouse-BOaTswain@users.noreply.github.com>
```

**Comment by deckhouse-BOaTswain** [comment]
New tag 'v1.30.11' is created.

**Comment by deckhouse-BOaTswain** [comment]
New commit [f33b5a](https://github.com/deckhouse/deckhouse/commit/f33b5a747234bd624745a7ce42996ad419c79a62) in branch 'main':
```
[prometheus] Exposing API doc fixes (#870)

Co-authored-by: Artem Kladov <artem.gen.kladov@gmail.com>
```
